client/src/containers/interventions/new/step2/supplier/component.tsx

```diff
@@ -173,13 +187,17 @@ const Supplier: FC = () => {
             <div className="mt-1">
               <Select
                 {...register('newLocationCountryInput')}
+                showSearch
                 loading={isLoadingCountries}
                 current={optionsCountries.find(
                   (option) => option.value === watch('newLocationCountryInput'),
                 )}
                 options={optionsCountries}
                 placeholder="All Countries"
                 onChange={(value) => handleDropdown('newLocationCountryInput', value)}
+                onSearch={(searchedCountry) =>
+                  handleDropdownSearch('newLocationCountryInput', searchedCountry)
+                }
```

Comment:
If I'm not wrong this `onSearch` is not needed, by default the search works. This is only needed when you want different behavior for the search.

Reply:
You are 100% right. Did not notice this, so now the `handleDropdownSearch ` is useless. Will remove and clean. Thanks!